config/initializers/spree.rb

```diff
@@ -18,6 +19,13 @@
   config.shipping_instructions = true
   config.address_requires_state = true
 
+  # Set Spree::Config defaults for testing
+  if Rails.env.test?
+    config.stripe_connect_enabled = false
+    config.enable_embedded_shopfronts = false
+    config.products_require_tax_category = false
+  end
```

Comment:
@sauloperez this stuff can't go in `test.rb` because `Spree::Config` is set in an initializer, so it doesn't exist yet. I will think about a better place to put them before this is merged.